splunk-otel-android/src/main/java/com/splunk/rum/RumScreenName.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * This annotation can be used to customize the {@code screen.name} attribute for an instrumented
- * Fragment or Activity. TODO: Deprecate this so the otel one can be used instead
+ * Fragment or Activity.
+ *
+ * @deprecated RumScreenName moved to io.opentelemetry.rum.internal.instrumentation package
```

Comment:
nit: Startcap to `@Deprecated`

Reply:
Changing to uppercase D breaks the tag on my local environment, even though the official android developer reference shows it uppercase. Do you know why that might be?

Reply:
@dvernon-splunk  `@Deprecated` is an annotation, what you are using is a javadoc tag which is supposed to be lowercase

Reply:
Thank you @laurit!